### PR TITLE
fixed census directory link

### PR DESCRIPTION
**Pull request by mattlevitan**
My name is Matt and I'm a data science student working with HMDA data. This repo is leaps and bounds ahead of cfpb/hmda-tools. Thanks for the legwork, hope I can contribute more.

### CODE REVIEW

File: load_scripts/SQL/create_and_load_lar_2004.sql [hunk 1]
```diff
@@ -47,7 +47,7 @@ CREATE TEMPORARY TABLE lar_load
 
 COPY lar_load
 -- Change this path to your local data path.
-FROM '{data_path}HMDA_Data_Science_Kit/data/lar/lar_2004.dat';
+FROM '/Users/mattlevitan/HMDA_Data_Science_Kit/data/lar/lar_2004.dat';
```
Comment (Kibrael) [inline load_scripts/SQL/create_and_load_lar_2004.sql]: Could you run [this script](https://github.com/cfpb/HMDA_Data_Science_Kit/blob/master/load_scripts/reset_path.py) to reset the paths? I should be able to merge these in after.

Thanks!

**Comment by Kibrael** [comment]
Hello @matthewlevitan, thank you for the contribution!  

I added a comment about data paths in the SQL script, let me know if you have questions.

File: load_scripts/SQL/create_and_load_lar_2004.sql (hunk 1 above)
Comment (Kibrael) [reply load_scripts/SQL/create_and_load_lar_2004.sql]: We had another pull request that made our data paths more useful. 

If you could re-pull the codebase and recommit the .sql files, I think we can get your PR merged.

Comment (mattlevitan) [reply load_scripts/SQL/create_and_load_lar_2004.sql]: Oops. didnt mean to push all three of those commits. what I get for working in master I believe my .sql paths are now fixed

Comment (Kibrael) [reply load_scripts/SQL/create_and_load_lar_2004.sql]: I think we've all made extra work for ourselves by developing on master.

Another PR was merged that affects some of the files you have in your commit. Can you pull down those changes to fix the conflict? Thanks so much!

**Pull request merged**